langchain/package.json

```diff
@@ -993,9 +993,9 @@
     "@elastic/elasticsearch": "^8.4.0",
     "@getmetal/metal-sdk": "*",
     "@getzep/zep-js": "^0.9.0",
-    "@gomomento/sdk": "^1.47.1",
-    "@gomomento/sdk-core": "^1.47.1",
-    "@gomomento/sdk-web": "^1.47.1",
+    "@gomomento/sdk": "^1.51.1",
+    "@gomomento/sdk-core": "^1.51.1",
+    "@gomomento/sdk-web": "^1.51.1",
```

Comment:
Bumped to get `searchAndFetchVectors` support in Momento client. `@gomomento/sdk` has the actual client implementation for Node.js; `@gomomento/sdk-web` for the browser client; and `sdk-core` for the interface and message types (needed to get the project to build).